modular_skyrat/modules/customization/modules/language/_foreigner_helper.dm

```diff
@@ -0,0 +1,8 @@
+// Forces foreigner to give non-human races their alternate language, if they have one, otherwise, give them uncommon, because we don't follow TG's species language stuff.
+/datum/quirk/foreigner/add()
+	if(iscarbon(quirk_holder) && quirk_holder.client)
+		var/datum/language_holder/language_holder = quirk_holder.get_language_holder()
+		var/language = quirk_holder.client.prefs.try_get_common_language()
```

Comment:
Since clients are fickle beings, let's do something like this so they get uncommon.
```suggestion
		var/language = quirk_holder.client?.prefs?.try_get_common_language()
		language = language ? language : /datum/language/uncommon
```